### PR TITLE
Removed cache condition for cohort exploring

### LINKED ISSUE #39: Dont cache too much in front when showing cohorts

### PR DESCRIPTION
**Pull request by BPierrick**
## Description

This removes the buggy behavior preventing a cohort to refresh its data if the user didn't change exploring subject (perimeter, patients, cohort,...)

## Fixes

- Fixes #39

### CODE REVIEW

File: src/state/exploredCohort.ts [hunk 1]
```diff
@@ -39,9 +39,6 @@ const fetchExploredCohort = createAsyncThunk<
   const stateCohort = state.exploredCohort.cohort
   let shouldRefreshData = true
   switch (context) {
-    case 'cohort':
-      shouldRefreshData = !stateCohort || Array.isArray(stateCohort) || stateCohort.id !== id
-      break
```
Comment (simonvadee) [inline src/state/exploredCohort.ts]: will this be a problem for aphp ?

**Review by simonvadee — APPROVED** [review]

File: src/state/exploredCohort.ts (hunk 1 above)
Comment (BPierrick) [reply src/state/exploredCohort.ts]: I guess yes

Comment (BPierrick) [reply src/state/exploredCohort.ts]: Do you wish to check app context for this ?

Comment (simonvadee) [reply src/state/exploredCohort.ts]: either a context check or a //FIXME (arkhn) comment to explain why we want to get rid of this line

**Pull request merged**